src/main/java/uk/gov/register/presentation/ListValue.java

```diff
@@ -21,7 +22,7 @@ public boolean isList() {
     @Override
     @JsonIgnore
     public String getValue() {
-        throw new UnsupportedOperationException();
+        return "[ " + String.join(", ", Lists.transform(elements, FieldValue::getValue)) + " ]";
```

Comment:
I don't think it's this object's job to render html - and in particular, this won't render the LinkValues as links